docs/pt-br/_summary.md

```diff
@@ -0,0 +1,121 @@
+* [Complete Newbs Guide](newbs.md)
```

Comment:
All of the links in this file need to have the language prefix added, or clicking them will kick back to the English docs.

_e.g._

```suggestion
* [Complete Newbs Guide](pt-br/newbs.md)
```